aiida/post-init.sh

```diff
@@ -1,47 +1,64 @@
 #!/bin/bash
 
-# Create aiida_config.yaml configuration file for activate-aiida
-
 # Parse first/last name from GIT_AUTHOR_NAME
 # See https://stackoverflow.com/a/17841619/1069467
-function join_by { local d=$1; shift; local f=$1; shift; printf %s "$f" "${@/#/$d}"; }
-user=($GIT_AUTHOR_NAME)
-first_name=${user[0]}
-last_name=`join_by ' ' "${user[@]:1}"`
+function join_by {
```

Comment:
I think you should just combine the changes from #78 in here - unless there is a good reason not to do it?

Reply:
I separated them because, in principle, the changes in #78 fixed the broken setup, even without updating the whole AiiDA setup (which was what we originally thought was required). Nonetheless, I went ahead and updated everything to keep up with the active AiiDA development since then. For me, it's also fine to just merge both changes into one PR, though.

Reply:
One reason to separate it and merge the other one first is that we provide a direct link on the Materials Cloud Archive website to a prepopulated RenkuLab project, like here:
![image](https://github.com/SwissDataScienceCenter/contributed-project-templates/assets/41700727/55725496-0ec4-445d-991f-661c979ca698)

As the link points to this repo, it could be already fixed by just the merge of #78, even without the upgrading of the AiiDA version. That being said, I'd like to finalize this effort here soon, so if merging takes only a few more hours/days, we can also hold off and just merge this PR here.

Reply:
Yeah feel free to do it incrementally by all means. btw, I can send you the renku frog for a nicer icon in MC ;)

Reply:
Feel free to share the frog with me and I'll send it to the MC people :D